### DISCUSSION TITLE
File Read Error

### DISCUSSION THREAD

**Discussion by zbadshah** [PHPOffice/PhpSpreadsheet#4013] [Q&A]
I am facing issue for reading the .xlsx file
PhpOffice
 \ 
PhpSpreadsheet
 \ 
Reader
 \ 
Exception
PHP 8.1.25
9.52.16
File "https://accounts.p-link.net/public/reports/Templates/Balance_Sheet_Template.xlsx" does not exist.

The url when posted directly works. So it is strange that it is failing. Anyone please advise. Thanks

**Comment by oleibman** [comment]
Attempting to access the cited url results in a failure for me:
```
Secure Connection Failed

An error occurred during a connection to accounts.p-link.net. SSL received a record that exceeded the maximum permissible length.

Error code: SSL_ERROR_RX_RECORD_TOO_LONG

    The page you are trying to view cannot be shown because the authenticity of the received data could not be verified.
    Please contact the website owners to inform them of this problem.
```
No reason to think it would work any better for PhpSpreadsheet than it does for a browser.

**Comment by zbadshah** [comment]
Thank you.

 

I could not see this error. Where did you find this error? I want to know for future so I don’t bother you again for such trivial things.

 

BR

 

Syed

**Reply by oleibman** [reply]
I entered the url into a browser. If it matters, it was Firefox on Windows 11.

**Reply by zbadshah** [reply]
This was probably because of permission. I will open the permission and you can try it. The link is opening for me.
I checked with cloud company and they investigated SSL and they found all okay.
Why would a simple READ call will fail with a NOT FOUND error, while we know that the file is there? I have spent hours finding clues but could not find any. It may not be related to Phpspreadsheet.

Let me know please if you have any ideas on checking issues or debugging this issue. Thanks

**Comment by oleibman** [comment]
Link still doesn't work for me. Same symptom as before. It also doesn't work in Edge, nor in Chrome.

**Reply by zbadshah** [reply]
Strange. However, the issue is resolved. It was internal permission issue. My blunder. I really want to thank you for giving attention to my problem. I feel ashamed that I wasted your time due to an oversight on my part.

Best regards
